Gemfile

```diff
@@ -42,14 +40,14 @@ gem 'whenever', require: false
 gem 'kaminari-bootstrap'
 gem 'six'
 gem 'amazon-ecs'
-#gem 'paperclip'
-#gem 'paperclip-googledrive'
+# gem 'paperclip'
+# gem 'paperclip-googledrive'
 gem 'holder_rails'
 gem 'pg_search', '~> 2.1.2'
 gem 'nokogiri', '~> 1.10'
 
 group :development, :test do
-  gem 'faker', '1.0.1'
+  gem 'ffaker'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.